pynet/datasets/lpsnc.py

```diff
@@ -0,0 +1,206 @@
+# -*- coding: utf-8 -*-
+##########################################################################
+# NSAp - Copyright (C) CEA, 2019
+# Distributed under the terms of the CeCILL-B license, as published by
+# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+# for details.
+# @Author: Zhou Kai
+# @GitHub: https://github.com/athon2
+# @Date: 2018-11-30 09:53:44
+##########################################################################
+
+"""
+Module that provides functions to prepare the lpsnc dataset.
+"""
+
+# Imports
+from collections import namedtuple
+import os
+import logging
+import numpy as np
+import pandas as pd
+import nibabel as nib
+import progressbar
+from pynet.datasets import Fetchers
+
+# Global parameters
+MODALITIES = ('T2w', 'T1w', 'ce-GADOLINIUM_T1w', 'FLAIR')
+Item = namedtuple("Item", ["input_path", "output_path", "metadata_path"])
+logger = logging.getLogger("pynet")
+
+
+# datasetdir="/neurospin/radiomics/workspace_LPSNC/LPSNC_data"
```

Comment:
Set this as a default value of the function.
I think I prefer modality=None to select all modalities.